tfbuilder/builder.go

```diff
@@ -26,6 +26,28 @@ type Builder struct {
 	policies         map[string]*PolicyBuilder
 }
 
+type ModifyMeshBuilder interface {
+	OnlyBuild
+	AddressableBuilder
+	AddMesh(mesh *MeshBuilder) *Builder
```

Comment:
@lobkovilya I did as you suggested on slack:

```
type AddPolicyBuilder interface {
	AddPolicy(builder *PolicyBuilder) *Builder
	Build() string
}
```

but this has a flaw of exposing the "powerful" type `Builder` after you call `AddMesh` so you can still modify internals after that 🤷 . I think I'd need to have a wrapper around `builder` to just restrict that. Do you want me to do that?